kernel/configs/README.md

```diff
@@ -0,0 +1,28 @@
+## How to use defconfig files
+
+defconfig files must be copied in the kernel source code directory at `arch/x86/configs/`
+and then using `make` command the `.config` file can be generated.
+
+For example
+
+```
+cp x86_kata_kvm_4.14.x_defconfig linux-4.14.22/arch/x86/configs/
+popd linux-4.14.22
+make ARCH=x86_64 x86_kata_kvm_4.14.x_defconfig
+make -j4
+```
+
+## How to modify defconfig files
+
+first the `.config` file must be generated and edit the `.config` using `make` command. The defconfig file
```

Comment:
Capitalize and I suggest the following rewrite (lines 17-18): Generate and edit the '.config'  using the 'make' command. The defconfig file is generated using `make savedefconfig`.